client/components/google-my-business/stats-nudge.js

```diff
@@ -0,0 +1,34 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+
+export const StatsNudge = ( { translate } ) => {
+	return (
+		<Card className="google-my-business__stats-nudge">
+			<h2 className="google-my-business__stats-nudge-heading">
+				{ translate( 'Is your business listed on Google?' ) }
+			</h2>
+
+			<p className="google-my-business__stats-nudge-description">
+				{ translate( 'Google my business etc.' ) }
+			</p>
+
+			<Button className="google-my-business__stats-nudge-button">
+				{ translate( 'Drive More Traffic' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Drive traffic to your blog' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).